functions/handlers/books.js

```diff
@@ -127,7 +129,7 @@ exports.commentOnBook = (req, res) => {
     createdAt: new Date().toISOString(),
     bookId: req.params.bookId,
     userHandle: req.user.handle,
-    userImage: req.user.imageUrl,
```

Comment:
same comment as above. When commenting on a book if we don't carry over userImage, we can't display the userImage whene we're rending in frontend